### ISSUE TITLE
Laser pointer (feature request)

### ISSUE THREAD

**Issue by jhschenker** [edemaine/cocreate#113]
This is a great interface.  Very useful!

A laser pointer option would be a great addition (and I imagine very easy to implement).    Right now, you can see a pointer if you hover an active stylus over the interface, but if you accidently touch you draw.  Being able to point without worrying about drawing (or selecting) would be great!
 
Jeff Schenker

**Comment by edemaine** [comment]
Safety-wise, this sounds like the pan tool (<img width="18" src="https://user-images.githubusercontent.com/2218736/95117039-e7d2ed80-0715-11eb-9168-98a299239cff.png">) but maybe with a different icon?

**Comment by jhschenker** [comment]
Yes that sounds about right. Pan tool would be the safest at present.  A "pan tool" that didn't do anything when you clicked would do this.

**Comment by edemaine** [comment]
A tool that does nothing sounds a little weird. Maybe a laser pointer tool, when clicked, could make some "self-destructing ink" -- show a highlight that disappears after a few seconds.  No permanent effect but nicer for pointing at things.

**Comment by jhschenker** [comment]
Indeed.  That makes much more sense!

**Comment by ndattani** [comment]
I very much love to use the laser pointer feature in PowerPoint. The mouse cursor just becomes a red laser-like image. It would be great if that could be implemented here too!

**Comment by edemaine** [comment]
Making a tool that just shows a dot of a specified color would be easy; is this of sufficient interest by itself?

The self-destructing ink seems more useful but also harder to implement, as we'd need to resolve the following:

* Do laser pointer strokes get added in the history? I think ideally yes. If no, they could be wedged into the presence information I guess.
* Does the server implement the destruction? Or the client using the pointer? Or all clients, locally? The pointer client destructing is simplest, but also a little dangerous: if they disappear, they'll never clean up their stroke. But maybe that's OK, as long as it can be deleted? I could see actually wanting highlights to stick around until the next highlight gets done...
* Does the laser pointer share all the properties (width and color) with the pen and other drawing tools, or does it have its own "current" settings? Related also to highlighter.
